.ado/templates/react-native-init.yml

```diff
@@ -27,27 +27,28 @@ steps:
       script: yarn build
       workingDirectory: vnext
 
-  # yarn ends up copying the whole node_modules folder when doing an install of a file package
-  # Delete node_modules, so that resolution is more like when installing from a published npm package
-  - task: CmdLine@2
-    displayName: Remove node_modules
+  - task: PowerShell@2
+    displayName: Launch test npm server (verdaccio)
     inputs:
-      script: rd /S /Q node_modules
-      workingDirectory: vnext
+      targetType: inline
+      script: |
+        start-process npx -ArgumentList @('verdaccio', '--config', './.ado/verdaccio/config.yaml')
 
-  # Remove directory ignored by npm
-  - task: CmdLine@2
-    displayName: Remove RNTesterCopy
-    inputs:
-      script: rd /S /Q RNTesterCopy
-      workingDirectory: vnext
+  - script: |
+      npm set registry http://localhost:4873
```

Comment:
>npm set registry http://localhost:4873 [](start = 6, length = 38)

Do you need to reset the registry back to normal after the test is complete?  I am wondering if the test machine will go back into the pool configured this way.

Reply:
The machines are reset between runs.  
https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/agents?view=azure-devops&tabs=browser
```
Each time you run a pipeline, you get a fresh virtual machine. The virtual machine is discarded after one use.
```